recipe/meta.yaml

```
@@ -1,44 +1,53 @@
 {% set name = "keras-tuner" %}
-{% set version = "1.2.1" %}
+{% set version = "1.4.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 21478f73e6f1ce5a34e8b378eb10dc935a5e6e6eb74deb46131cd23653d13422
+  sha256: 6befd25ee81476e6207d8ca7ed7dc674b8194437cfa0b127294cd00da905ff22
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  # No tensorflow for python 3.13
+  skip: True  # [py<37 or py>=313]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
+    - pytorch  # [osx]
   run:
-    - python >=3.7
-    - Ipython
-    - kt-legacy
-    - numpy
+    - python
+    - keras
     - packaging
     - requests
+    - kt-legacy
+    - tensorflow >=2.0
```

```suggestion
    - tensorflow >=2.17
```
The mkl variant gets preferred. However we stopped having one after 2.12. So it solves an old 2.12. Then the pip check fails with tf 2.12. Likely a packaging issue with tf 2.12.
Given how old is our 2.12 release, setting the bound to >=2.17 should be sufficient.

---

Thank you!